### ISSUE TITLE
Nuget Packages missing description

### ISSUE THREAD

**Issue by jzabroski** [rsocket/rsocket-net#2]
- https://www.nuget.org/packages/RSocket.Core/
- https://www.nuget.org/packages/RSocket.Rpc.Core/

**Comment by jzabroski** [comment]
It appears the fix should be here: https://github.com/rsocket/rsocket-net/blob/master/RSocket.Core/RSocket.Core.csproj

See: https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli and also https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packagedescription

It appears you want to modify the csproj to add:

```xml
<PackageId>RSocket.Core</PackageId>
<PackageDescription>RSocket.NET is the RSocket protocol for Reactive Streams socket programming in .NET</PackageDescription>
<Authors>Robert Roeser</Authors>
<Company>Netifi</Company>
```

**Comment by robertroeser** [comment]
@rdegnan @dbarnett-netifi ?

**Comment by OlegDokuka** [comment]
fixed as of 0.2.7 release
